.gitignore

```diff
@@ -2,4 +2,11 @@ dist/
 coverage/
 node_modules/
 npm-debug.log
-.nyc_output
\ No newline at end of file
+.nyc_output
+
+# Output of running npm pack for testing
+*.tgz
+
+# Ignore exported helpers, but include src
+helpers/
+!src/helpers
```

Comment:
You don't end your files with a blank line? What savagery is this?

Reply:
😭 Fixing